.circleci/config.yml

```diff
@@ -160,77 +160,77 @@ workflows:
       - check-code-formatting
       - build-and-test
       - assume-role-development:
-          context: api-assume-role-development-context
+          context: api-assume-role-housing-development-context
           requires:
             - build-and-test
           filters:
             branches:
-              only: development
+              only: master
       - terraform-init-and-apply-to-development:
           requires:
             - assume-role-development
           filters:
             branches:
-              only: development
+              only: master
       - deploy-to-development:
           requires:
             - assume-role-development
           filters:
             branches:
-              only: development
-#   check-and-deploy-staging-and-production:
-#       jobs:
-#       - build-and-test:
-#           filters:
-#             branches:
-#               only: master
-#       - assume-role-staging:
-#           context: api-assume-role-staging-context
-#           requires:
-#               - build-and-test
-#           filters:
-#              branches:
-#                only: master
-#       - terraform-init-and-apply-to-staging:
-#           requires:
-#             - assume-role-staging
-#           filters:
-#             branches:
-#               only: master
-#       - deploy-to-staging:
-#           requires:
-#             - assume-role-staging
-#           filters:
-#             branches:
-#               only: master
-#       - permit-production-terraform-release:
-#           type: approval
-#           requires:
-#             - deploy-to-staging
-#       - assume-role-production:
-#           context: api-assume-role-production-context
-#           requires:
-#               - permit-production-terraform-release
-#           filters:
-#              branches:
-#                only: master
-#       - terraform-init-and-apply-to-production:
-#           requires:
-#             - assume-role-production
-#           filters:
-#             branches:
-#               only: master
-#       - permit-production-release:
-#           type: approval
-#           requires:
-#             - deploy-to-staging
-#           filters:
-#             branches:
-#               only: master
-#       - deploy-to-production:
-#           requires:
-#             - permit-production-release
-#             - assume-role-production
-#           filters:
-#             branches:
-#               only: master
+              only: master
+  check-and-deploy-staging-and-production:
+      jobs:
+      - build-and-test:
```

Comment:
Please include the 'check-code-formatting' job here as well (same as in the deploy to development workflow) :)

Reply:
Done! :)